ruby-challenges.rb

```diff
@@ -10,7 +10,23 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# Create a method called even_or_odd.
+# Use conditional statement to iterate.
+# Expected output: 7 is odd.
+# Expected output: 42 is even.
+# Expected output: 221 is odd.
 
+def even_or_odd num
+  if num % 2 == 0
+    "#{num} is even"
+  elsif num % 2 != 0
+    "#{num} is odd"
+  end
+end
```

Comment:
Logic looks good.  Ruby also has built-in methods called .even? and .odd? that will perform the evaluation for you.